webpack/dev.config.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const env = require('node-env-file')
+const path = require('path')
+const webpack = require('webpack')
+
+const config = require('./config')
+const { contentBase,
+  host,
+  url
+} = require('./server.config.js')
+
+const secretDir = path.join(__dirname, '../secret.sh')
+if (!fs.existsSync(secretDir)) {
```

Comment:
Can we allow settings via env vars too?

Sthg like : `!fs.existsSync(secretDir) || (!env. STRIPE_PUBLISHABLE_KEY && !env. GOOGLE_MAPS_API_KEY)

Reply:
👍